crates/bin/pcli/src/command/validator.rs

```diff
@@ -107,6 +112,9 @@ pub enum DefinitionCmd {
         /// definition may be generated using the `pcli validator definition sign` command.
         #[clap(long)]
         signature: Option<String>,
+        /// The selected fee tier to multiply the fee amount by.
+        #[clap(short, long, value_enum, default_value_t)]
```

Comment:
Remove the `value_enum` clap attribute throughout. Again, see #4539 for context.